hash/apple2_flop_orig.xml

```diff
@@ -28052,161 +27535,481 @@ license:CC0-1.0
+		<info name="version" value="1.0" />
+		<!--Modified to remove identifying site license information.-->
+		<sharedfeat name="compatibility" value="A2E,A2EE,A2C,A2GS" />
+		<!--Dump released: 2022-02-25-->
+		<!--educational program-->
+		<part name="flop1" interface="floppy_3_5">
+			<dataarea name="flop" size="1296218">
+				<rom name="cryptoquest 800k.woz" size="1296218" crc="08cf7ada" sha1="310fc8b1db2e827fb5678e08de4271699fffd7fe" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="cryscavr">
+		<description>Crystal Caverns</description>
+		<year>1982</year>
+		<publisher>Hayden Book Company</publisher>
+		<info name="developer" value="Imaginative Systems" />
+		<info name="programmer" value="Daniel Kitchen" />
+		<info name="usage" value="Requires a 48K Apple ][ or later." />
+		<sharedfeat name="compatibility" value="A2,A2P,A2E,A2EE,A2C,A2GS" />
+		<!--Dump released: 2022-02-11-->
+		<!--adventure game-->
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="234793">
+				<rom name="crystal caverns.woz" size="234793" crc="c0ea16f3" sha1="0101b207d85530d16c74b1a6eb52420cfcc0761a" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="cyborg2">
+		<description>Cyborg (version 2)</description>
+		<year>1981</year>
+		<publisher>Sentient Software</publisher>
+		<info name="programmer" value="Michael Berlyn" />
+		<info name="usage" value="Requires a 48K Apple ][+ or later." />
+		<info name="version" value="2" />
+		<sharedfeat name="compatibility" value="A2P,A2E,A2EE,A2C,A2GS" />
+		<!--Dump released: 2022-02-25-->
+		<!--adventure game-->
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="234793">
+				<rom name="cyborg v2.woz" size="234793" crc="69539113" sha1="4668ba1edb2be92a423f66297e62ac4dee0c9e82" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="dqcarmndalm">
+		<description>Dataquest: Carmen's North Dakota Almanac Database (version 1.0)</description>
+		<year>1989</year>
+		<publisher>North Dakota Database Project</publisher>
+		<info name="developer" value="North Dakota Database Project" />
+		<info name="usage" value="Requires a 128K Apple //e or later." />
+		<info name="version" value="1.0" />
+		<sharedfeat name="compatibility" value="A2E,A2EE,A2C,A2GS" />
+		<!--Dump released: 2020-03-11-->
+		<!--educational program-->
+		<!--This is part of the Where in North Dakota is Carmen Sandiego disk set. Please refer to the following website for more details on the history of this particular disk:
+		https://web.archive.org/web/https://gamehistory.org/where-in-north-dakota-is-carmen-sandiego/
+		-->
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="234788">
+				<rom name="carmen's north dakota almanac database.woz" size="234788" crc="757d3e67" sha1="618423e72b222f16e6624750d80e09aecf6f4656" />
+			</dataarea>
+		</part>
+	</software>
```

Comment:
Were these ever sold separately, or only as part of the “Where in North Dakota is Carmen Sandiego” package?  (This and the rest of the North Dakota Database Project stuff.)  If the latter is the case, they should be parts of that item.

(That would potentially fall over if there were updates to the Database Project disks released independently of the game, as we don’t have a way of having parent/clone parts within an item.)

Reply:
@cuavas i think you are right.  According to the link, the game + disks were released as a set to school systems in ND. There's no indication that the dataquest disks were sold separately or updated. I can make this change.  
https://gamehistory.org/where-in-north-dakota-is-carmen-sandiego/

Reply:
@cuavas There are a couple of issues in merging the disks into a single entry for Where in North Dakota is Carmen Sandiego.  1.  The MECC Dataquest databases have different hardware requirements than the game.  Dataquest: "Requires a 128K Apple //e or later.", where the Game: "Requires a 64K Apple ][+ or later.".  2.  "North Dakota Cities Database (version 1.6)" is version 1.6 and is noticeably different than the other databases.   Being version 1.6 may indicate that we could find additional versions in the future.  Please let me know if you want me to continue to merge the entries, and I'll do it this weekend.   If I merge them, I will change the game hardware requirements to an //e.

Thanks, 
A-Noid

Reply:
If the items are combined, I’d keep the requirements of the combined item as the minimum required to play the game itself, and add a note that the Dataquest disks have higher requirements.

Is 1.6 the version of the Dataquest program on the disk?  I wonder if there’s something about the Cities database that required an upgrade to the program?  We don’t seem to have dumps of multiple versions of any of the disks though.

I’ll merge this PR for now – it’s no worse than before anyway.